src/Prelude.hs

```diff
@@ -1,8 +0,0 @@
--- | Uses [universum](https://hackage.haskell.org/package/universum) as default Prelude.
-
-module Prelude
-       ( module Universum
```

Comment:
I guess it's fine to use `base-noprelude` and keep this module replacing `Universum` to `Relude` 🙂 
It allows not to add `import Relude` line to each file.